features/step_definitions/group_selection_steps.rb

```diff
@@ -352,8 +361,16 @@
   expect(page).to have_content("Covered: #{primary.first_name}", wait: 10)
 end
 
+Then(/^.* should see primary and dependent person on enrollment$/) do
+  primary = Person.all.select { |person| person.primary_family.present? }.first
+  dependent = Person.all.select { |person| person.primary_family.blank? }.first
+
+  expect(page).to have_content("Covered: #{primary.first_name} #{dependent.first_name}", wait: 10)
```

Comment:
Same feedback as the other comments-- can we avoid hardcoding the English string here by targeting some other attribute or by relying on an existing translation? I realize there is an existing example that already has the hardcoded string, but that is a pattern we don't want to repeat.